front/pages/api/w/[wId]/data_sources/[name]/index.ts

```diff
@@ -110,6 +111,7 @@ async function handler(
       const ds = await dataSource.update({
         description,
         visibility: req.body.visibility,
+        userUpsertable: req.body.userUpsertable,
```

Comment:
not exactly related, but there's nothing preventing people from using this endpoint on a managed DS right ?

Reply:
Fair point. I'll add a check :+1:

Reply:
(done)